app/Repositories/Metrics/MetricsRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Repositories\Metrics;
+
+use App\User;
+use App\Models\Playlist;
+use App\Models\Activity;
+use App\Models\ActivityMetric;
+
+class MetricsRepository
```

Comment:
Need to enhance Repository as mentioned in previous comment

Reply:
Discussion in MembershipRepository comment